sql_assignment1_create_tables.sql

```diff
@@ -0,0 +1,116 @@
+/******create 'student_db' database*****/
+create DATABASE student_db;
+
+/******switch to database*****/
+USE student_db;
+
+/******create table school*****/
+create table school(
+    id int PRIMARy KEY,
+    name varchar(20),
+    address varchar(50),
+    principal varchar (20),
+    phone int
```

Comment:
try modifying this field with varchar data type. Commit the required queries to modify it

Reply:
Updated the phone datatype by using ALTER TABLE at the end of the file.